OsmAnd/src/net/osmand/plus/plugins/aistracker/AisSimulationProvider.java

```diff
@@ -25,7 +25,7 @@ public AisSimulationProvider(@NonNull AisTrackerPlugin plugin) {
 	public void startSimulation(@NonNull File file) {
 		AisTrackerLayer layer = plugin.getLayer();
 		if (layer != null) {
-			AisMessageListener listener = new AisMessageListener(layer, file);
+			AisMessageListener listener = new AisMessageListener(layer, file, 100);
```

Comment:
Make constant

Reply:
Created AisMessageSimulationListener with delay. 100 is constant